execution/executor/src/components/apply_chunk_output.rs

```diff
@@ -285,7 +286,7 @@ impl ApplyChunkOutput {
         ))
     }
 
-    fn assemble_ledger_diff(
+    pub fn assemble_ledger_diff(
```

Comment:
I guess you don't need to make this pub to use in test?

Reply:
fixed